<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机图像采集</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid gray;</string>
      </property>
      <property name="text">
       <string>相机图像显示区域</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="controlLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="deviceGroupBox">
        <property name="title">
         <string>设备控制</string>
        </property>
        <layout class="QVBoxLayout" name="deviceLayout">
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="text">
            <string>刷新设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deviceComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>连接相机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>断开连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="captureGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>图像采集</string>
        </property>
        <layout class="QVBoxLayout" name="captureLayout">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>开始采集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>停止采集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="snapButton">
           <property name="text">
            <string>单帧采集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>保存图像</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="statusGroupBox">
        <property name="title">
         <string>状态信息</string>
        </property>
        <layout class="QVBoxLayout" name="statusLayout"/>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="statusTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
